### PR TITLE
Fixing bugs

### PR DESCRIPTION
**Pull request by hania-imtisal234**
1) Fixing: Redirect to Home for "/" path
2) Fixing: Fix collapse button to the bottom.
3) Fixing: Navbar height should be screen.
4) Fixing: Back button on all countries page should be working properly
5) Fixing:  Selected filter in all countries should be highlighted.

### CODE REVIEW

File: src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -12,146 +31,8 @@ button {
...
-  margin-top: 240px;
 }
 
-.div-button {
-  margin-bottom: 5px;
-}
 .arrow {
   color: #f6f4eb;
 }
-
```
Comment (usamahmed97) [inline src/Components/Sidebar/Sidebar.css:24]: Is this all related to selection?

Comment (hania-imtisal234) [reply src/Components/Sidebar/Sidebar.css:24]: No it's the arrow icon color

Comment (usamahmed97) [reply src/Components/Sidebar/Sidebar.css:24]: I mean the whole CSS that is removed.

Comment (hania-imtisal234) [reply src/Components/Sidebar/Sidebar.css:24]: No it's for collapsed and expanded sidebar

**Review by usamahmed97 — APPROVED** [review]
